File: XlabSalesApiCore/Models/Invoice.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace XlabSales.Models
{
    public partial class Invoice
    {
        public Invoice()
        {
            InvoiceDetails = new HashSet<InvoiceDetail>();
        }

        public int Id { get; set; }
        public string ClientName { get; set; }
        public DateTime? Date { get; set; }

        public virtual ICollection<InvoiceDetail> InvoiceDetails { get; set; }
    }
}

File: XlabSalesAngular/src/app/models/Invoice.ts
import { InvoiceDetails } from "./InvoiceDetails";

export class Invoice{
    id: number;
    clientName : string;
    date: string;
    invoiceDetails : InvoiceDetails[];

    constructor(){
        this.id = 0;
        this.clientName = '';
        this.date = '';
        this.invoiceDetails =[];
    }
}